mypy.ini

```diff
@@ -19,4 +19,3 @@ disallow_untyped_defs = false
 [mypy-homeassistant.util.yaml]
 warn_return_any = false
 disallow_untyped_calls = false
-
```

Comment:
Revert this change (add the blank line back).

Reply:
Fixed in this [commit](https://github.com/home-assistant/home-assistant/pull/22325/commits/cb7ab1110a369f7e24762a61cbd82e59e47106b4).
:-)